jython/MakeAllWindowsVisible.py

```diff
@@ -0,0 +1,527 @@
+#   The "UnHide Panels" script was designed to cope with a Windows OS problem described in the lengthy 
```

Comment:
Maybe change this line as well. Otherwise a user might be confused about what the "UnHide Panels" script are.

Reply:
Done, thanks.